docs/pages/installation/kubernetes.md

```diff
@@ -5,6 +5,15 @@ permalink: installation/kubernetes
 ---
 
 # Quick Start with Kubernetes
+## <strong>Using Helm</strong>
+<strong>Helm v3 - run the following:</strong>
+
+```
+  $ git clone https://github.com/layer5io/meshery.git; cd meshery
+  $ kubectl create namespace meshery
+  $ helm install meshery --namespace meshery install/kubernetes/helm/meshery
+```
+## <strong>Using Kubernetes cluster</strong>
```

Comment:
I have a concern about this line, Should we change it to `## <strong>Deploying by YAML file</strong>`. I mean either Helm or YAML all deploy meshery on Kubernetes. So, `Using Kubernetes cluster ` may not make sense.